charts/spire/charts/spiffe-oidc-discovery-provider/values.yaml

```diff
@@ -88,10 +88,46 @@ livenessProbe:
 ## @param podAnnotations [object] Pod annotations for Spire OIDC discovery provider
 podAnnotations: {}
 
-insecureScheme:
-  ## @param insecureScheme.enabled Flag to enable insecure schema
-  enabled: false
+tls:
+  spire:
+    ## @param tls.spire.enabled Use spire to secure the oidc-discovery-provider
+    enabled: true
+
+  externalSecret:
+    ## @param tls.externalSecret.enabled Provide your own certificate/key via tls style Kubernetes Secret
+    enabled: false
+    ## @param tls.externalSecret.secretName Specify which secret to use
+    secretName: ""
 
+  certManager:
+    ## @param tls.certManager.enabled Use certificateManager to create the certificate
+    enabled: false
+    issuer:
+      ## @param tls.certManager.issuer.create Create an issuer to use to issue the certificate
+      create: true
+      acme:
+        ## @param tls.certManager.issuer.acme.email Must be set in order to register with LetsEncrypt. By setting, you agree to their Terms of Service
+        email: ""
+        ## @param tls.certManager.issuer.acme.server Server to use to get certificate. Defaults to LetsEncrypt
+        server: https://acme-v02.api.letsencrypt.org/directory
+        # Testing server: https://acme-staging-v02.api.letsencrypt.org/directory
+        ## @param tls.certManager.issuer.acme.solvers [object] Configure the issuer solvers. Defaults to http01 via ingress.
+        solvers: {}
+        # - http01:
+        #     ingress:
+        #       ingressClassName: nginx
+    certificate:
+      ## @param tls.certManager.certificate.dnsNames Override the dnsNames on the certificate request. Defaults to the same settings as Ingress
```

Comment:
Is there value in making this function like additional dns names?

Reply:
Letsencrypt in some configurations supports wildcard certificates. So instead of using the default certificate request, you may want to override it all and just set
```
dnsNames:
- *.example.com
```